company-profiles/travis-ci.md

```diff
@@ -2,10 +2,29 @@
 
 ## Company blurb
 
-⚠ We don't have much information about this company yet!
+Test and Deploy with Confidence
+Easily sync your projects with Travis CI and you’ll be testing your code in minutes!
 
-If you know something we don't, help us fill it in!  Here's how:
+## Company size
 
-- Read our [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md)
-- Have a look at our [example company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md)
-- Follow the structure of the example profile and [send us a pull request with your changes to this file!](https://github.com/remoteintech/remote-jobs/edit/master/company-profiles/travis-ci.md)
+11-50 employees 
+
+## Region
+* Europe
+* Asia
+* North America
+
+### region 1
```

Comment:
Either worldwide or certain regions as above?

Reply:
Its seems they are worldwide since they have people from Russia, India, Nigeria, Germany and so on. 
I will fix this mark .